notification-portlet-webcomponents/notification-modal/src/components/NotificationModal.vue

```diff
@@ -96,10 +96,26 @@ export default {
           }
         );
 
+        var unreadNotifications = [];
+
+        for (var i = 0; i < notifications.length; i++) {
```

Comment:
:art:
```js
const unreadNotifications = notifications.filter(
  n => !(n.attributes && n.attributes.READ && n.attributes.READ[0] === "true")
);
```

Or if you want to add https://babeljs.io/docs/en/next/babel-plugin-proposal-optional-chaining.html

```js
const unreadNotifications = notifications.filter(n => !(n?.attributes?.READ[0] === "true"));
```

Reply:
I'll make the change. Thanks for the suggestion